doc/md/16-language-manual.md

```diff
@@ -690,46 +690,46 @@ As bitwise types, these types support bitwise operations and (`&`), or (`|`) and
 
 As bitwise types, these types support bitwise operations and (`&`), or (`|`) and exclusive-or (`^`). Further, they can be rotated left (`<<>`), right (`<>>`), and shifted left (`<<`), right (`>>`). The right-shift is logical. All shift and rotate amounts are considered modulo the number’s bit width *n*.
 
-The corresponding module in the base package provides conversion functions:
+The corresponding module in the core package provides conversion functions:
 
-- Conversion to [`Nat`](https://internetcomputer.org/docs/motoko/base/Nat.md).
+- Conversion to [`Nat`](https://internetcomputer.org/docs/motoko/core/Nat.md).
 
-- Checked and wrapping conversions from [`Nat`](https://internetcomputer.org/docs/motoko/base/Nat.md).
+- Checked and wrapping conversions from [`Nat`](https://internetcomputer.org/docs/motoko/core/Nat.md).
 
 - Wrapping conversion to the bounded, signed integer type of the same size.
 
-### Type [`Blob`](https://internetcomputer.org/docs/motoko/base/Blob.md)
+### Type [`Blob`](https://internetcomputer.org/docs/motoko/core/Blob.md)
 
-The type [`Blob`](https://internetcomputer.org/docs/motoko/base/Blob.md) of category O (Ordered) represents binary blobs or sequences of bytes. Function `b.size()` returns the number of characters in [`Blob`](https://internetcomputer.org/docs/motoko/base/Blob.md) value `b`. Operations on blob values include sequential iteration over bytes via function `b.values()` as in `for (v : Nat8 in b.values()) { …​ v …​ }`.
+The type [`Blob`](https://internetcomputer.org/docs/motoko/core/Blob.md) of category O (Ordered) represents binary blobs or sequences of bytes. Function `b.size()` returns the number of characters in [`Blob`](https://internetcomputer.org/docs/motoko/core/Blob.md) value `b`. Operations on blob values include sequential iteration over bytes via function `b.values()` as in `for (v : Nat8 in b.values()) { …​ v …​ }`.
 
-### Type [`Principal`](https://internetcomputer.org/docs/motoko/base/Principal.md)
+### Type [`Principal`](https://internetcomputer.org/docs/motoko/core/Principal.md)
 
-The type [`Principal`](https://internetcomputer.org/docs/motoko/base/Principal.md) of category O (Ordered) represents opaque principals such as canisters and users that can be used to identify callers of shared functions and used for simple authentication. Although opaque, principals may be converted to binary [`Blob`](https://internetcomputer.org/docs/motoko/base/Blob.md) values for more efficient hashing and other applications.
+The type [`Principal`](https://internetcomputer.org/docs/motoko/core/Principal.md) of category O (Ordered) represents opaque principals such as canisters and users that can be used to identify callers of shared functions and used for simple authentication. Although opaque, principals may be converted to binary [`Blob`](https://internetcomputer.org/docs/motoko/core/Blob.md) values for more efficient hashing and other applications.
```

Comment:
Do we have a way of automatically checking that every link is correct (does not return 404 or another error)?

Reply:
This is covered for relative links through Docusaurus. We might eventually want to convert these URLs to relative links or see there is a prebuilt way to verify these as well.